core-galleon-pack/src/main/resources/feature_groups/host-profile.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="host-profile" xmlns="urn:jboss:galleon:feature-group:1.0">
+    <feature-group name="jmx"/>
```

Comment:
Putting this after elytron is the traditional order.

Reply:
ok, thanks!